lua/hbac/utils.lua

```diff
@@ -10,6 +7,8 @@ M.get_listed_buffers = function()
 end
 
 M.buf_autoclosable = function(bufnr)
+	local config = require("hbac.config")
+	local state = require("hbac.state")
```

Comment:
Is there a reason to move it here?

Reply:
No you're right, I think this is a leftover from when I was trying to debug the require-loop, which seems to only be an issue in hbac.command.actions. I'll check but otherwise I'll put this back